{{cookiecutter.project_slug}}/webpack.config.js

```diff
@@ -85,37 +86,30 @@ module.exports = {
       filename: '[name].css',
     }),
     new SpriteLoaderPlugin(),
+    new CleanWebpackPlugin({
+      cleanOnceBeforeBuildPatterns: ['**/*', '!VERSION'],
```

Comment:
I guess we can use the bare plugin without options, we don't have any `VERSION` file in the output folder.